common/cards/alter-egos/hermits/renbob-rare.ts

```diff
@@ -56,6 +58,14 @@ class RenbobRare extends Card {
 					query.row.index(component.slot.row.index),
 				)?.entity || null,
 			)
+			attack.updateLog((values) => {
+				if (values.previousLog === undefined) return ''
+				return values.target === EMPTY_ROW
+					? values.previousLog
+							.replace(`${values.target} `, '')
+							.replace(`for ${values.damage} damage`, 'and missed')
+					: values.previousLog
+			})
```

Comment:
I don't think this is the right approach. It would be better to check if the attack doesn't have a target  and write a log for that case instead of cleaning up for the other broken stuff.